integration/testcmd/mountkexec/uinit/mountkexec.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"log"
+
+	"github.com/u-root/u-root/pkg/sh"
+	webboot "github.com/u-root/webboot/pkg/mountkexec"
+	"golang.org/x/sys/unix"
+)
+
+// Mount a vfat volume and kexec the kernel within.
+func main() {
+
+	if err := webboot.MountISO("/dev/sda1", ""); err != nil {
```

Comment:
why is it /dev/sda1?

Reply:
dev/sda1 is a vfat file created in intergration,go.I am using it to test my mountISO since I dont have an ISO yet.I will replace it with an actucal iso once I make one.